src/odemis/driver/xt_client.py

```diff
@@ -1447,6 +1458,24 @@ def _cancelAutoContrastBrightness(self, future):
     #         return True
 
 
+class FibScanner(model.Emitter):
+    """
+    This is an extension of the model.Emitter class for controlling the FIB. Currently via XTLib only minimal control
+    of the FIB is possible, which is limited to pause/unpausing the image beam, and getting the latest image (via the
+    detector with the appropriate channel).
+    """
+
+    def __init__(self, name, role, parent, **kwargs):
+        model.Emitter.__init__(self, name, role, parent=parent, **kwargs)
```

Comment:
I'd suggest to have a argument "channel", with the default value "ion2".
This value should be store on the .channel (or .channel_name) attribute .